modules/phase_field/doc/content/modules/phase_field/Initial_Conditions.md

```diff
@@ -0,0 +1,166 @@
+# Initial Conditions
+
+With the phase field method, the initial condition is critical, as it establishes the initial microstructure of the material. MOOSE has a system for creating [Initial Conditions (ICs)](/syntax/ICs/index.html), however there are various initial conditions that have been created as part of the Phase Field Module. The corresponding .C files for each IC can be found in `moose/modules/phase_field/src/ics` except for BoundingBoxIC which is found in `moose/framework/src/ics`.
+
+## BoundingBoxIC
+
+!media /phase_field/BoundingBoxIC.png  style=width:200px;padding-left:20px;float:right;
+    caption=BoundingBoxIC
+
+BoundingBoxIC allows setting the initial condition of a value inside and outside of a specified box. The box is aligned with the x,y,z axis, and is specified by passing in the x,y,z coordinates of the bottom left point and the top right point. Each of the coordinates of the "bottom_left" point MUST be less than those coordinates in the "top_right" point. When setting the initial condition if bottom_left <= Point <= top_right then the `inside` value is used. Otherwise the `outside` value is used. More information can be found on the [BoundingBoxIC](/BoundingBoxIC.md) syntax page.
+
+### Inputs
+
+- `x1`: The x coordinate of the lower left-hand corner of the box
+- `y1`: The y coordinate of the lower left-hand corner of the box
+- `z1`: The z coordinate of the lower left-hand corner of the box
+- `x2`: The x coordinate of the upper right-hand corner of the box
+- `y2`: The y coordinate of the upper right-hand corner of the box
+- `z2`: The z coordinate of the upper right-hand corner of the box
+- `inside`: The value of the variable inside the box
+- `outside`: The value of the variable outside the box
```

Comment:
Should the argument list (this object and others) be removed and replaced with a link to the syntax for the individual objects?